src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/test/WattTimeDataSourceTests.cs

```diff
@@ -247,6 +247,52 @@ public void GetCurrentCarbonIntensityForecastAsync_ThrowsWhenTooFewDatapointsRet
         Assert.ThrowsAsync<WattTimeClientException>(async () => await this.DataSource.GetCurrentCarbonIntensityForecastAsync(location));
     }
 
+    [TestCase("2022-01-01T00:00:00Z", "2022-01-01T00:00:00Z", TestName = "Not Rounded")]
```

Comment:
```suggestion
    [TestCase("2022-01-01T00:00:00Z", "2022-01-01T00:00:00Z", TestName = "GetCarbonIntensityForecastAsync returns expected value: requestedAt == floor")]
```

We've been starting to add prefixes to testcase names so they show up better in test runner output.  Considering adding something like ^ to these names